docs/source/get-started/training.rst

```diff
@@ -209,14 +209,26 @@ as follows passing as ``callbacks`` the classes :py:class:`~kornia.x.ModelCheckp
 
 .. code:: python
 
+    def my_evaluate(self):
+      # stats = StatsTracker()
```

Comment:
add a comment or something to illustrate what users could do

Reply:
Added more verbose comment below but I'm not sure if I should add more to my_evaluate method or whole callback?